apps/tailwind-components/components/form/FieldInput.vue

```diff
@@ -58,10 +63,26 @@ function validate(value: columnValue) {
     :id="id"
     :label="label"
     :required="required"
-    :value="data as string"
+    :value="(data as string)"
     @focus="$emit('focus')"
     @update:modelValue="$emit('update:modelValue', $event)"
     @error="$emit('error', $event)"
   ></LazyInputTextArea>
+  <LazyInputListbox
```

Comment:
I think there will be multiple ways to display a ref so for now I would be okay to merge like this but there are already other PRs in the making for the other representation. As written above. 

My proposal is to make explicit in emx metadata if you like to display a checkbox/radio/select/tree or table.